internal/app/commands/cloud/client/constants.go

```diff
@@ -13,7 +13,11 @@ var (
 )
 
 func apiURL(path string) string {
-	url := os.Getenv(CLI_CLOUD_API_URL)
+	url := "https://cloud.enonic.com/api"
```

Comment:
I'd created a const for that next to CLI_CLOUD_APU_URL, this way it will be easier to spot next time

Reply:
Fixed